lib/workos/webhooks.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+# typed: true
+
+require 'json'
+require 'openssl'
+
+module WorkOS
+  # The Webhooks module provides convenience methods for working with the WorkOS webhooks.
+  # You'll need to extract the signature header and payload from the webhook request
+  # sig_header = request.headers['WorkOS-Signature']
```

Comment:
Maybe an extra newline in this comment to separate the code example from the documentation text.